shared/src/main/scala/pl/metastack/metatime/Constants.scala

```diff
@@ -0,0 +1,28 @@
+package pl.metastack.metatime
+
+object Constants {
+    final val YEAR_INDEX = 1
```

Comment:
Can you write `YearIndex`, `MonthIndex`, etc.?

The indentation is off. It should be only two characters.